src/partials/editor.html

```diff
@@ -55,6 +55,9 @@ <h5 class="section-heading">Layers</h5>
                 <label class="gf-form-label width-4">Min</label>
                 <input type="text" class="gf-form-input width-10 ng-pristine ng-untouched ng-valid ng-not-empty" ng-change="ctrl.render()" ng-model="ctrl.panel.min[$index]">
 
+		<label class="gf-form-label width-4">Size</label>
```

Comment:
When size is not given, make it such that the default lineWidth is 3. This is because having a default lineWidth less than 3 wouldn't show twin-tubes clearly as well as it becomes difficult hovering over the thin lines.

Reply:
Im looking at this right now and it seems to be already initializing to a size 3. Maybe It should be set up to a higher default value?

Reply:
I have tested the panel and it seems like the default lineWidth is definitely less than 3. We need it to be a default width of 3 when the map loads/when the size is not specified in the editor